docs/ldml/tr35-info.md

```diff
@@ -38,35 +38,35 @@ The LDML specification is divided into the following parts:
 
 ## <a name="Contents" href="#Contents">Contents of Part 6, Supplemental</a>
 
-*   1 [Introduction Supplemental Data](#Supplemental_Data)
-*   2 [Territory Data](#Territory_Data)
-    *   2.1 [Supplemental Territory Containment](#Supplemental_Territory_Containment)
-    *   2.2 [Subdivision Containment](#Subdivision_Containment)
-    *   2.3 [Supplemental Territory Information](#Supplemental_Territory_Information)
-    *   2.4 [Territory-Based Preferences](#Territory_Based_Preferences)
-        *   2.4.1 [Preferred Units for Specific Usages](#Preferred_Units_For_Usage)
-            *   Table: [Unit Preference](#Unit_Preferences)
-    *   2.5 [\<rgScope>: Scope of the “rg” Locale Key](#rgScope)
-*   3 [Supplemental Language Data](#Supplemental_Language_Data)
-    *   3.1 [Supplemental Language Grouping](#Supplemental_Language_Grouping)
-*   4 [Supplemental Code Mapping](#Supplemental_Code_Mapping)
-*   5 [Telephone Code Data](#Telephone_Code_Data) (Deprecated)
-*   6 [Postal Code Validation (Deprecated)](#Postal_Code_Validation)
-*   7 [Supplemental Character Fallback Data](#Supplemental_Character_Fallback_Data)
-*   8 [Coverage Levels](#Coverage_Levels)
-    *   8.1 [Definitions](#Coverage_Level_Definitions)
-    *   8.2 [Data Requirements](#Coverage_Level_Data_Requirements)
-    *   8.3 [Default Values](#Coverage_Level_Default_Values)
-*   9 [Supplemental Metadata](#Appendix_Supplemental_Metadata)
-    *   9.1 [Supplemental Alias Information](#Supplemental_Alias_Information)
-        *   Table: [Alias Attribute Values](#Alias_Attribute_Values)
-    *   9.2 [Supplemental Deprecated Information (Deprecated)](#Supplemental_Deprecated_Information)
-    *   9.3 [Default Content](#Default_Content)
-*   10 [Locale Metadata Elements](#Metadata_Elements)
-*   11 [Version Information](#Version_Information)
-*   12 [Parent Locales](#Parent_Locales)
-*   13 [Unit Conversion](#Unit_Conversion)
-*   14 [Unit Preferences](#Unit_Preferences)
+* 1 Introduction [Supplemental Data](#Supplemental_Data)
+* 2 [Territory Data](#Territory_Data)
+  * 2.1 [Supplemental Territory Containment](#Supplemental_Territory_Containment)
+  * 2.2 [Subdivision Containment](#Subdivision_Containment)
+  * 2.3 [Supplemental Territory Information](#Supplemental_Territory_Information)
+  * 2.4 [Territory-Based Preferences](#Territory_Based_Preferences)
+    * 2.4.1 [Preferred Units for Specific Usages](#Preferred_Units_For_Usage)
```

Comment:
The entry for the Table after 2.4.1 is missing here

Reply:
It's a table caption. I'll see if I can use an option to bring it back.

Reply:
Still missing?

Reply:
There's actually no table there! Maybe it used to be there.  It reads like this:

```md
#### 2.4.1 <a name="Preferred_Units_For_Usage" href="#Preferred_Units_For_Usage">Preferred Units for Specific Usages</a>

_For information about preferred units and unit conversion, see Section 13 [Unit Conversion](#Unit_Conversion) and Section 14 [Unit Preferences](#Unit_Preferences)._

### 2.5 <a name="rgScope" href="#rgScope">`<rgScope>`: Scope of the “rg” Locale Key</a>
```

So no table between 2.4.1 and 2.5 … maybe it was moved… 

if we are otherwise happy with this, and this PR is uncovering errors/inconsistencies, i propose merging this and then fixing the actual text afterwards.

Reply:
Agree...